oauth2_http/java/com/google/auth/oauth2/OAuthException.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020, Google Inc. All rights reserved.
```

Comment:
This is the old style copyright. It's LLC now. @chingor13 Can you confirm the exact format we're using?

Reply:
The copyright line should be:
```suggestion
 * Copyright 2020 Google LLC
```

The rest should be the same as the BSD license used throughout this repo.